src/components/VaultParamChange.tsx

```diff
@@ -0,0 +1,216 @@
+import type { Amount } from '@agoric/ertp/src/types';
+import { Fragment } from 'react';
+import { FiChevronDown } from 'react-icons/fi';
+import clsx from 'clsx';
+import { Menu, Transition } from '@headlessui/react';
+import { motion } from 'framer-motion';
+import {
+  LoadStatus,
+  usePublishedDatum,
+  usePublishedKeys,
+  WalletContext,
+} from 'lib/wallet';
+import { useContext, useState } from 'react';
+import { AmountInput, PercentageInput } from './inputs';
+
+interface Props {
+  charterOfferId: number;
+}
+
+export type ParameterValue =
+  | {
+      type: 'invitation';
+      value: Amount<'set'>;
+    }
+  | {
+      type: 'ratio';
+      value: { numerator: Amount<'nat'>; denominator: Amount<'nat'> };
+    }
+  | {
+      type: 'amount';
+      value: Amount;
+    };
+
+export default function VaultParamChange(props: Props) {
+  const walletUtils = useContext(WalletContext);
+  const { data: vaultKeys, status: vaultKeysStatus } =
+    usePublishedKeys('vaultFactory');
+
+  const managerIds = vaultKeys.filter(key => key.startsWith('manager'));
+  const [managerId, setManagerId] = useState(null);
+
+  const { data, status } = usePublishedDatum(
+    `vaultFactory.${managerId}.governance`,
+  );
+
+  const { data: selectedManagerMetrics } = usePublishedDatum(
+    `vaultFactory.${managerId}.metrics`,
+  );
+
+  const collateralBrand = selectedManagerMetrics?.totalCollateral?.brand;
+
+  // "Alleged: IbcATOM brand" -> "IbcATOM"
+  const collateralBrandLabel = collateralBrand?.iface?.split(' ')[1];
+
+  const [minutesUntilClose, setMinutesUntilClose] = useState(10);
+
+  const [paramPatch, setParamPatch] = useState({});
+
+  console.log('ProposeParamChange', { data, paramPatch });
+
+  const canMakeProposal = !!props.charterOfferId && collateralBrand;
```

Comment:
if you want it boolean:
```suggestion
  const canMakeProposal = !!(props.charterOfferId && collateralBrand);
```

Reply:
Done thanks